flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "monpad";
```

Comment:
I had hoped to make this totally generic (remove `./haskell` directory specifier, `release` flag etc.), and use it across other projects. `Build.hs` would then just pull it down when necessary from some repository (with an integrity check). But it seems that [flake files have to be checked in](https://github.com/NixOS/nix/issues/6642), so that's a no-go.

Reply:
Anyway, once we upload to Hackage (#20), the package will be picked up by Haskell.nix, all assets will be in the sdist, and we won't need to do anything special in Nix at all. Although perhaps it would be useful to retain the ability to deploy binaries without a corresponding Hackage version?